WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -1140,4 +1140,41 @@ decisions = {
 			}
 		}
 	}
+	divide_roman_empire = {
+		only_playable = yes
+		is_high_prio = yes
+		potential = {
+			has_landed_title = e_roman_empire
+			age = 16
+			realm_size = 200
+			tier = emperor
```

Comment:
This is a redundant condition.

Reply:
Which one? tier = emperor? Yes, not needed.

On the previous comment, C option is designed for children, so the condition will break it.

Reply:
It works nonetheless. I tested it with your build to reproduce the issue. 

And `tier = emperor` is indeed the one I replied to.

Reply:
How does it work? Who becomes the second child when there's only one?

Reply:
The next successor in line, or at least that is what it did for me.

Reply:
In theory it could work when your only child wasn't a heir, and another dynasty member was a heir.
I will fix it.

Reply:
You can set the target to any dynasty member.

Reply:
But any_dynasty_member includes non-children.

Reply:
Yes, and that is on purpose, because your heirs aren't necessarily children.

Reply:
But the option IS about children 🤔

Reply:
Why does it need to be about children? I don't understand your insistance when heirs don't need to be childrens.

Reply:
- This option is designed to emulate the historical division by Theodosius.
- There in no scope for characters second and further in succession.

Reply:
1. I thought the point was to implement the suggestion, not to reproduce Theodosius' divide to the letter, hence my concern for expanding it.
2. You can check for heirs, the other child can be a monk anyway.

Reply:
1. That could be done with a new option.
2. Good point on the possibility of children being monks. I will fix the option trigger to prevent them from gaining the imperial title.

Reply:
- Yes, if it is impossible to do it in one why not?

Reply:
It will be easier to do this in separate option because of localisation and infallibility.